docs/UserGuide.md

```diff
@@ -14,25 +14,19 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `MyInsuRec.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
```

Comment:
Is our jar file in `https://github.com/se-edu/addressbook-level3/releases`?

Reply:
no, cmi it should be <https://github.com/AY2223S1-CS2103T-W16-4/tp/releases>